src/test/plugins/ucc_discovery_plugin_test.cpp

```diff
@@ -1,3 +1,5 @@
+#include <gtest/gtest.h>
```

Comment:
This is a false positive, third party heades should not be above STL headers. Usually, we don't care about IWYU in tests and do not include the gtest header.

@Bouncner what's your take here? Just remove the include or add it to the third-party libraries in the include order?